app/swagger/requests/bb/health_records.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+module Swagger
+  module Requests
+    module BB
+      class HealthRecords
+        include Swagger::Blocks
+
+        swagger_path '/v0/health_records/refresh' do
+          operation :get do
+            key :description, 'Retrieves patient status'
+            key :operationId, 'bbHealthRecordsRefresh'
+            key :tags, %w(bb health-records refresh)
+
+            response 200 do
```

Comment:
I think there is at least one 422 error class that we might be want to test here. Though I think we could possibly revisit this as well seeing as those validations are currently disabled pending a F/E change.